pkg/cmd/plugincheck2/main.go

```diff
@@ -28,17 +30,32 @@ type FormattedOutput struct {
 
 func main() {
 	var (
-		strictFlag    = flag.Bool("strict", false, "If set, plugincheck returns non-zero exit code for warnings")
+		strictFlag = flag.Bool(
+			"strict",
+			false,
+			"If set, plugincheck returns non-zero exit code for warnings",
+		)
 		configFlag    = flag.String("config", "", "Path to configuration file")
-		sourceCodeUri = flag.String("sourceCodeUri", "", "URL to the source code of the plugin. If set, the source code will be downloaded and analyzed. This can be a ZIP file or an URL to git repository")
+		sourceCodeUri = flag.String(
+			"sourceCodeUri",
+			"",
+			"URL to the source code of the plugin. If set, the source code will be downloaded and analyzed. This can be a ZIP file or an URL to git repository",
+		)
+		checksum = flag.String(
+			"checksum",
+			"",
+			"checksum of the plugin archive. MD5 or SHA1. A string with the hash or an url to a file with the hash",
```

Comment:
Nit: I would make this a bit more clear:

```suggestion
			"checksum of the plugin archive. MD5, SHA1 or a string with the hash or an url to a file with the hash",
```